backend/webapp/settings.py

```diff
@@ -138,3 +138,7 @@
 # Use to override the OPTIONS.driver value in the database settings
 # (eg, if django is using a different driver to ogr2ogr)
 OGR2OGR_DRIVER = None
+
+
+MEDIA_ROOT = 'media/'
+MEDIA_URL = 'media/'
```

Comment:
These bits make me nervous -- I _think_ they should already be set or have default values, but also it looks like your'e treating them as the same thing.  MEDIA_ROOT is the disk/storage location, while MEDIA_URL is the server endpoint. (Having typed all that you're probably doing the right thing, it just looks odd and worth commenting on)

Reply:
Actually on deploying I found an issue -- it assumes that basemap is in the media directory, but it isn't (at least, not the production configured media root)